docs/primers/tables.md

```diff
@@ -9,6 +9,264 @@ Tablespecs
+These are run *after* all of the CBD-based values ('fields', 'joins', and 'counts') has been gathered, and the 'computed_fields'
+property can *only* exist at the base level of the table specification.
+
+Computed fields *must* contain 'fieldName' and 'value' properties and 'value' must be a valid computed field function
+ specification.  Optionally, they may also contain a 'property' property.
+
+Computed fields are run in the order they appear in the specification, so a later fields can use the results of earlier functions.
+
+Functions can be nested, so the input values of a function can be supplied by another function, e.g.:
+
+```javascript
+{
+    ...
+    "computed_fields": [
+        {
+            "fieldName": "foo",
+            "value": {
+                "conditional" : // This defines the function to run
+                {
+                    "if:[
+                        {
+                            "arithmetic": // This will run an arithmetic function to set the left side of the condition
+                            {
+                                ["$x", "*", 10]
+                            }
+                        }, ">", 100
+                    ],
+                    "then": "foo",
+                    "else": "bar"
+                }
+            }
+
+        }
+    ]
+}
+```
+
+Only *one* function can be defined at a time, however.  This is *not* valid:
+
+```javascript
+{
+    ...
+    "computed_fields": [
+        {
+            "fieldName": "foo",
+            "value": {
+                "replace": {
+                    "search": "foo",
+                    "replace": "bar",
+                    "subject": "$x"
+                },
+                "arithmetic":["$x", "-", 4"]
+            }
+        }
+    ]
+}
+```
+
+Fields are referenced as variables defined by the fieldName prefixed with ``` $ ```: ``` "fieldName": "foo" ``` would be ``` $foo ```, etc.
+
+Conditional functions
+---------------------
+
+Conditionals let you set the value of the table field based on other values.  Conditionals *must* contain an 'if' property
+and must have either a 'then' or an 'else' property, or both.
+
+"if" expects an array with a length of 1 or 3 as its value: if the array's count is 1, it will just check for the existence
+of the value:
+
+```javascript
+{
+    "fieldName": "foo",
+    "value": {
+        "conditional": {
+            "if": ["$x"] // Does the fieldName 'x' have a value?
+            "then": true,
+            "else": false
+        }
+    }
+}
+```
+
+If the array's count is three, the 2nd value must be one of:
+">","<",">=", "<=", "==", "!=", "contains", "not contains", "~=", "!~"
+there is no strict type comparison.
+
+"then" and "else" values may be of any type, including another function.  To replicate 'else if', you can use another
+ conditional function as the value of 'else'.
+
+ Arithmetic functions
```

Comment:
Do you want something in here to tell you what arithmetic functions are available?